### ISSUE TITLE
Reverse Even Iterator

### ISSUE THREAD

**Issue by blond** [blond/ho-iter#4]
Returns a Iterator, that traverses iterators in reverse order.

This is reminiscent of traverses of several arrays in reverse order.

**Example:**

``` js
const reverseEvenly = require('ho-iter').reverseEvenly;

const arr1 = [1, 2];
const arr2 = [3, 4];

const set1 = new Set([1, 2]);
const set2 = new Set([3, 4]);

for (let i = arr1.length - 1; i >= 0; i--) {
    console.log(arr2[i]);
    console.log(arr1[i]);
}

// 4, 2, 3, 1

for (let item of reverseEvenly(set1, set2)) { console.log(item) } // 4 2 3 1
```

**Comment by blond** [comment]
If you need reverse some iterators you can use `reverse` with `evenly` methods.

**Example:**

``` js
const eventy = require('ho-iter').eventy;
const reverse = require('ho-iter').reverse;

const set1 = new Set([1, 2]);
const set2 = new Set([3, 4]);

const iter = eventy(set1, set2); // 1, 3, 2, 4
const reversedIter = reverse(iter); // 4, 2, 3, 1
```

**Comment by blond** [comment]
Closed favor of #10
